imap_processing/spice/geometry.py

```diff
@@ -73,12 +73,12 @@ class SpiceFrame(IntEnum):
     SpiceFrame.IMAP_ULTRA_45: np.array([0, 0, 1]),
     SpiceFrame.IMAP_ULTRA_90: np.array([0, 0, 1]),
     SpiceFrame.IMAP_MAG: np.array([0, 0, 1]),
-    SpiceFrame.IMAP_SWE: np.array([0, 0, 1]),
-    SpiceFrame.IMAP_SWAPI: np.array([0, 0, 1]),
+    SpiceFrame.IMAP_SWE: np.array([-1, 0, 0]),
+    SpiceFrame.IMAP_SWAPI: np.array([0, 1, 0]),
     SpiceFrame.IMAP_CODICE: np.array([0, 0, 1]),
```

Comment:
Can we remove above TODO and note that we need to update CoDICE only now?

Reply:
Yep, good catch on the TODO. We (@bourque , @sdhoyt , and I) met with Joey and the takeaway was that CoDICE boresight isn't needed until L3, so we will leave it as is for now. Joey didn't seem too sure, but that is how we left it.